qml/ApplicationMenus/MenuItem.qml

```diff
@@ -77,11 +77,11 @@ ActionItem {
     RowLayout {
         id: row
         spacing: units.gu(1)
-        anchors.left: parent.left
-        anchors.right: parent.right
+        anchors.left: root.left
+        anchors.right: root.right
         anchors.leftMargin: units.gu(1)
         anchors.rightMargin: units.gu(1)
-        anchors.verticalCenter: parent.verticalCenter
+        anchors.verticalCenter: root.verticalCenter
```

Comment:
This change (and the two above it) are not needed, and when using anchors it's preferable to use `parent`, than referencing by id. I would revert these.

Reply:
what was the warning? Does it go away if you add a binding `parent: root`?

Reply:
```
QWARN  : qmltestrunner::MenuBar::test_stray_submenus() file:///home/dalton/unity8/qml/ApplicationMenus/MenuItem.qml:80: TypeError: Cannot read property of null
QWARN  : qmltestrunner::MenuBar::test_stray_submenus() file:///home/dalton/unity8/qml/ApplicationMenus/MenuItem.qml:81: TypeError: Cannot read property of null
QWARN  : qmltestrunner::MenuBar::test_stray_submenus() file:///home/dalton/unity8/qml/ApplicationMenus/MenuItem.qml:84: TypeError: Cannot read property of null
```

I can log parent in `Component.onCompleted` and get `qml: MenuItem_QMLTYPE_12_QML_18(0x1d43d50, "menuBar-item0-menu-item1-actionItem")` as expected.

This does not go away with a `parent: root` binding.

Reply:
OK for your changes, but I wonder if we should file a bug on the uitk's ActionItem.